packages/cli/src/common/sandbox.ts

```diff
@@ -0,0 +1,29 @@
+import { copyFileSync, mkdirSync, readdirSync, rmdirSync } from 'fs'
+import * as path from 'path'
+
```

Comment:
The functions of this file are copied and pasted from `packages/framework-integration-tests/integration/helper/fileHelper.ts`, as I don't know how to share common functionality like this inside a lerna monorepo (without creating an extra package, of course).

Does anybody have an idea to share helper function accross packages in a lerna monorepo?

Reply:
Essentially creating a package :(

Reply:
If you want a 100% clean solution, these functions should be in a module that is imported from all the other packages anyway like `framework-types`, and renaming `framework-types` as `framework-common`. This is a refactor I'd love to do, because I've never liked the idea of separating types in a module just because they're  types...

But we're talking about code that is used both from production code (the CLI) and the integration tests. I think that we can safely assume that integration tests are going to be "always" executed in a context where you have access to this file, no matter where it is, so you can delete the copy in the integration test package and just import this with an extra serve of `'..'` in your path. As soon as the file is reachable it can be used and it's not a big deal to break the package structure from integration tests because any changes that make it stop working will be detected by the compiler anyway 😜

Reply:
Good suggestions, thanks guys! Maybe I try to create a private package called "common" or "util", because this is starting to occur many times. The problem with that is that the package will be published and it doesn't make much sense to publish this. 
I think I'll go with the Javi approach. Let's see

Reply:
Done here: 392efbd4d349b5132defa024aa3a968736b1a6ef